build.zig

```diff
@@ -154,7 +155,7 @@ pub fn build(b: *Builder) !void {
         .target = target,
     });
     tests.root_module.addImport("ssz", ssz);
-    tests.root_module.addImport("@zeam/types", zeam_types);
+    // tests.root_module.addImport("@zeam/types", zeam_types);
```

Comment:
temp change to run tests else it was complaining types redeclared or something. also brings up the issue of how to run tests of parent modules